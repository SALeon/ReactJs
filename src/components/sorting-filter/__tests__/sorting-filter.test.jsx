import React from 'react';
import { shallow } from 'enzyme';
import SortingFilter from '../index';

const mockSortingInfo = {
  label: 'Sort by',
  items: [{ id: '1', label: 'release' }, { id: '2', label: 'date' }, { id: '3', label: 'rating' }],
};

it('SortingFilter snapshot', () => {
  const wrapper = shallow(<SortingFilter sortingData={mockSortingInfo} />);
  expect(wrapper).toMatchSnapshot();
});
