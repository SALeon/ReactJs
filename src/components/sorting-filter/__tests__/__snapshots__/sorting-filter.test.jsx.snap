// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingFilter snapshot 1`] = `
<div
  className="sorting-filter"
>
  <span>
    Sort by
  </span>
  <ToggleList
    itemClassStyle="sorting-filter__item"
    itemContainerStyle="sorting-filter__item-container"
    items={
      Array [
        Object {
          "component": <button
            className="sorting-filter__item"
            type="button"
          >
            release
          </button>,
          "id": "1",
        },
        Object {
          "component": <button
            className="sorting-filter__item"
            type="button"
          >
            date
          </button>,
          "id": "2",
        },
        Object {
          "component": <button
            className="sorting-filter__item"
            type="button"
          >
            rating
          </button>,
          "id": "3",
        },
      ]
    }
    listClassStyle="sorting-filter__list"
  />
</div>
`;
